db/migrate/20160302170230_rename_digest_after_days_to_digest_after_minutes.rb

```diff
@@ -4,6 +4,7 @@ class RenameDigestAfterDaysToDigestAfterMinutes < ActiveRecord::Migration[4.2]
   def up
     rename_column :user_options, :digest_after_days, :digest_after_minutes
     execute "UPDATE user_options SET digest_after_minutes = digest_after_minutes * 1440 WHERE digest_after_minutes IS NOT NULL"
+    execute "DELETE FROM site_settings WHERE name = 'default_email_digest_frequency' AND value = ''"
```

Comment:
I'm not sure if this is the right fix here. It feels like the query after should avoid updating the value if `value = ''`

Reply:
Sure, that works as well. PR updated.